qemu/tests/expose_host_mtu.py

```diff
@@ -54,23 +32,25 @@ def cleanup_ovs_ports(netdst, ports):
         :param ports: existing ports need to be remain before this test
         """
 
-        if is_ovs_backend(netdst) is True:
-            ports = set(get_ovs_ports(netdst).splitlines()) - \
-                set(ports.splitlines())
+        host_bridge = utils_net.find_bridge_manager(netdst)
+        if utils_net.ovs_br_exists(netdst) is True:
+            ports = set(host_bridge.list_ports(netdst)) - set(ports)
             for p in ports:
-                process.system("ovs-vsctl del-port %s %s" % (netdst, p))
+                utils_net.find_bridge_manager(netdst).del_port(netdst, p)
 
     netdst = params.get("netdst", "switch")
+    host_bridge = utils_net.find_bridge_manager(netdst)
     if netdst in utils_net.Bridge().list_br():
         host_hw_interface = utils_net.Bridge().list_iface(netdst)[0]
-    elif is_ovs_backend(netdst) is True:
-        host_hw_interface = get_ovs_ports(netdst)
-        tmp_ports = re.findall(r"t[0-9]{1,}-[a-zA-Z0-9]{6}", host_hw_interface)
+    elif utils_net.ovs_br_exists(netdst) is True:
```

Comment:
@xiywang Here should use "utils_net.ovs_br_exists(ovs=netdst)", otherwise it's checking bridge type.